vdoctl/cmd/configure_matrix.go

```diff
@@ -77,6 +77,7 @@ var matrixConfigureCmd = &cobra.Command{
 			cobra.CheckErr(err)
 		}
 
+		fmt.Println("Tip: Compatibility matrix has been configured, you might want to try 'vdoctl deploy' to deploy VDO.")
```

Comment:
`Tip: Compatibility matrix has been configured, you might want to try 'vdoctl configure drivers' to configure Drivers.`

Flow is something like this
Deploy->Configure Matrix -> Configure Driver -> Check Status